src/js/hmm/components/HMMTaxonomy.tsx

```diff
@@ -35,7 +31,7 @@ export function HMMTaxonomy({ counts, title }: HMMTaxonomyProps) {
 
     const components = map(sorted, ({ name, count }) => (
         <HMMTaxonomyItem key={name}>
-            {name} <Badge>{count}</Badge>
+            {name} <Badge className="ml-auto">{count}</Badge>
         </HMMTaxonomyItem>
```

Comment:
Instead of apply a class to `Badge`, change `HMMTaxonomyItem` to use `flex` and `justify-between`.